alphalens/tears.py

```diff
@@ -693,13 +702,20 @@ def create_event_study_tear_sheet(factor_data,
             UserWarning
         )
 
-    for p in factor_returns:
 
+    if '1D' in factor_returns:
         plotting.plot_cumulative_returns(
-            factor_returns[p],
-            period=p,
+            factor_returns['1D'],
+            period='1D',
             freq=trading_calendar,
-            ax=gf.next_row()
+            ax=gf.next_row(),
+        )
+
+        plotting.plot_cumulative_returns(
+            factor_returns['1D'],
+            period='1D',
+            freq=trading_calendar,
+            ax=gf.next_row(),
```

Comment:
I would totally remove ` plotting.plot_cumulative_returns` call from `create_event_study_tear_sheet`. It is very likely the user will get wrong results.

Reply:
You're right. I'll remove it.